airbyte-webapp/src/components/LoadingSchema/LoadingSchema.tsx

```diff
@@ -19,7 +19,7 @@ const FetchMessage = styled.div`
 
 const LoadingSchema: React.FC = () => (
   <SpinnerBlock>
-    <Spinner />
+    <ProgressBar runTime={240} />
```

Comment:
Could we please leave an inline comment here in code how we got to this rather arbitrary number? Like that this is an experience based amount of time this task usually runs, but so we'll be able in the future to understand where this number was coming from.

Reply:
Hehe, I believe we need to ask @jrhizor for such info.

Reply:
This number just is purely a middle-of-the-road estimate based off of personal experiences running checks/discover across docker-compose OSS / kube OSS / cloud. In the future we can tune this based off of run durations when we actually capture them.